controllers/reconciler/proxy.go

```diff
@@ -13,17 +13,20 @@ import (
 )
 
 const (
+	nameProxy             = "proxy"
 	imageProxy            = "proxy"
 	proxyEnvConfig        = "NSM_CONFIG_MAP_NAME"
 	proxyEnvService       = "NSM_SERVICE_NAME"
-	proxyEnvVip           = "NSM_VIPS"
 	proxyEnvSubnetPools   = "NSM_SUBNET_POOLS"
 	proxyEnvSubnetLengths = "NSM_SUBNET_PREFIX_LENGTHS"
 	proxyEnvIpam          = "NSM_IPAM_SERVICE"
 	proxyEnvLb            = "NSM_NETWORK_SERVICE_NAME"
-	proxyEnvNsp           = "NSM_NSP_SERVICE"
```

Comment:
Should the proxy.yaml file be updated with deletion of these two envVars too?

Reply:
right, I will delete them